task3/script.js

```diff
@@ -1,15 +1,17 @@
 const ul = document.getElementById('ul');
+const liLength = 2;
 
-for (let i =1; i <=2; i++) {
+for (let i =1; i <= liLength; i++) {
     const li =document.createElement('li');
     const a =document.createElement('a');
     const img =document.createElement('img');
+    const data = ["bookmark.png", "message.png"];
```

Comment:
As for the data array, you can define it outside of the for statement.
If you define it inside the for statement, you will probably have to define the data array twice.

Reply:
I understand.
I have created an unnecessary array.

I corrected it, so please review it again.